front_end/html/view_answer.html

```diff
@@ -34,3 +46,28 @@ <h4>Instructor's comments</h4>
     <p>This exercise does not exist.</p>
 {% end %}
 <br /><br />
+
```

Comment:
I like that this makes the page more flexible and allows longer solutions to be displayed because they will be scrollable. However, I thinking it would be better to just use regular <code> or <pre> tags because the Ace boxes will not be editable, and it will prevent us from having to hit server for these Javascript files. But let me know if I'm missing something.